### PR TITLE
Add new functions

### PR DESCRIPTION
**Pull request by mets11rap**
Added dropTable, renameTable, and listTables. I also fixed the lockfile version.

### CODE REVIEW

File: src/QueryBuilder.js [hunk 1]
```diff
@@ -208,6 +208,39 @@ class QueryBuilder {
...
+    return this._knex.hasTable(table).then((exists) => {
+      if(exists) return
+      return this._knex.schema.renameTable(table, newname)
+    }).catch(this._logger.error)
+  }
+
+  /**
+   * @private
+   */
+  _listTables () {
```
Comment (mets11rap) [inline src/QueryBuilder.js]: I'm not exactly sure if this will work. I think it will though.

File: src/QueryBuilder.test.js [hunk 2]
```diff
@@ -86,3 +86,15 @@ test('update', (t) => {
...
+
+})
+
+test('renameTable', (t) => {
+
+})
+
+test('listTables', (t) => {
+  
+})
```
Comment (mets11rap) [inline src/QueryBuilder.test.js]: I was unable to write tests since I don't know how, sorry.

File: src/QueryBuilder.js [hunk 3]
```diff
@@ -208,6 +208,39 @@ class QueryBuilder {
...
+  }
+
+  /**
+   * @private
+   * @param   {String}             table      The name of the table.
+   * @param   {String}             newname    The new name of the table.
+   */
+  _renameTable ({ table, newname }) {
+    return this._knex.hasTable(table).then((exists) => {
+      if(exists) return
```
Comment (alex-taxiera) [inline src/QueryBuilder.js]: hi can you just put a space before () so if (exists)

File: src/QueryBuilder.js [hunk 4]
```diff
@@ -234,11 +235,12 @@ class QueryBuilder {
   }
 
   /**
+   * List all tables in the database
    * @private
    */
   _listTables () {
-    return this._knex.raw("select * from information_schema.tables where table_type='base table'")
-      .then((success) => 0)
+    return this._knex.raw("SELECT table_name FROM information_schema.tables WHERE table_schema='public' AND table_type='BASE TABLE';")
```
Comment (mets11rap) [inline src/QueryBuilder.js]: I fixed it and tested with a Heroku Postgres database. This may vary with other systems so I'd appreciate more tests.

**Pull request merged**